### PR TITLE
KB-5598 user pagination implementation

### PR DESCRIPTION
**Pull request by garima468**
KB-5598 user pagination implementation

### CODE REVIEW

File: project/ws/app/src/lib/routes/home/routes/users-view/users-view.component.ts [hunk 1]
```diff
@@ -82,159 +84,271 @@ export class UsersViewComponent implements OnInit {
...
+  inActiveUsers(query: string) {
+    this.loaderService.changeLoad.next(true)
+    const inactiveUsersData: any[] = []
+    const status = this.currentFilter === 'active' ? 1 : 0
+    this.currentOffset = this.limit * ((this.pageIndex + 1) - 1)
+    this.usersService.getAllKongUsersPaginated(this.rootOrgId, status, this.limit, this.currentOffset, query).subscribe(
+      (data: any) => {
+        this.userDataTotalCount = data.result.response.count
+        this.usersData = data.result.response
+        if (this.usersData && this.usersData.content && this.usersData.content.length > 0) {
```
Comment (vishnubansaltarento) [inline project/ws/app/src/lib/routes/home/routes/users-view/users-view.component.ts]: check for this.usersData.content.length as well then this.usersData.content.length>0

File: project/ws/app/src/lib/routes/home/routes/users-view/users-view.component.ts [hunk 2]
```diff
@@ -82,159 +84,271 @@ export class UsersViewComponent implements OnInit {
...
-          break
-        default:
-          this.data = usersData
-          break
-      }
-    }
-  }
-  newKongUser(active: boolean) {
-    // const rootOrgId = _.get(this.route.snapshot.parent, 'data.configService.unMappedUser.rootOrg.rootOrgId')
-    const usersData: any[] = []
```
Comment (vishnubansaltarento) [inline project/ws/app/src/lib/routes/home/routes/users-view/users-view.component.ts:219]: Why this whole code has been removed?

Comment (garima468) [reply project/ws/app/src/lib/routes/home/routes/users-view/users-view.component.ts:219]: implemented separate function for Active & Inactive users, this is not needed now.

File: project/ws/app/src/lib/routes/home/routes/users-view/users-view.component.ts (hunk 1 above)
Comment (garima468) [reply project/ws/app/src/lib/routes/home/routes/users-view/users-view.component.ts]: fixed this, and pushed

**Pull request merged**